### ISSUE TITLE
Unknown operation cvmfs-reload

### ISSUE THREAD

**Issue by DrDaveD** [cvmfs/cvmfs#3721]
I just tried an upgrade from cvmfs-2.11.5 to the testing release of cvmfs-2.12.0 and noticed a strange message:
```
Unknown operation cvmfs-reload.
```
Here's the full output:
```
$ sudo dnf --disablerepo=osg --enablerepo=cernvm-testing install cvmfs
No read/execute access in current directory, moving to /
CernVM testing packages                          27 kB/s |  56 kB     00:02
Package cvmfs-2.11.5-1.osg23.el8.x86_64 is already installed.
Dependencies resolved.
================================================================================
 Package           Architecture Version              Repository            Size
================================================================================
Upgrading:
 cvmfs             x86_64       2.12.0-1.el8         cernvm-testing        14 M
 cvmfs-fuse3       x86_64       2.12.0-1.el8         cernvm-testing        12 M
 cvmfs-libs        x86_64       2.12.0-1.el8         cernvm-testing        11 M

Transaction Summary
================================================================================
Upgrade  3 Packages

Total download size: 38 M
Is this ok [y/N]: y
Downloading Packages:
(1/3): cvmfs-libs-2.12.0-1.el8.x86_64.rpm       5.4 MB/s |  11 MB     00:02
(2/3): cvmfs-2.12.0-1.el8.x86_64.rpm            6.5 MB/s |  14 MB     00:02
(3/3): cvmfs-fuse3-2.12.0-1.el8.x86_64.rpm      3.2 MB/s |  12 MB     00:03
--------------------------------------------------------------------------------
Total                                           9.6 MB/s |  38 MB     00:03
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                        1/1
  Running scriptlet: cvmfs-libs-2.12.0-1.el8.x86_64                         1/1
  Upgrading        : cvmfs-libs-2.12.0-1.el8.x86_64                         1/6
  Running scriptlet: cvmfs-libs-2.12.0-1.el8.x86_64                         1/6
  Running scriptlet: cvmfs-2.12.0-1.el8.x86_64                              2/6
  Upgrading        : cvmfs-2.12.0-1.el8.x86_64                              2/6
  Running scriptlet: cvmfs-2.12.0-1.el8.x86_64                              2/6
Unknown operation cvmfs-reload.

  Upgrading        : cvmfs-fuse3-2.12.0-1.el8.x86_64                        3/6
  Running scriptlet: cvmfs-fuse3-2.12.0-1.el8.x86_64                        3/6
  Cleanup          : cvmfs-fuse3-2.11.5-1.osg23.el8.x86_64                  4/6
  Running scriptlet: cvmfs-2.11.5-1.osg23.el8.x86_64                        5/6
  Cleanup          : cvmfs-2.11.5-1.osg23.el8.x86_64                        5/6
  Running scriptlet: cvmfs-2.11.5-1.osg23.el8.x86_64                        5/6
  Cleanup          : cvmfs-libs-2.11.5-1.osg23.el8.x86_64                   6/6
  Running scriptlet: cvmfs-libs-2.11.5-1.osg23.el8.x86_64                   6/6
  Verifying        : cvmfs-2.12.0-1.el8.x86_64                              1/6
  Verifying        : cvmfs-2.11.5-1.osg23.el8.x86_64                        2/6
  Verifying        : cvmfs-fuse3-2.12.0-1.el8.x86_64                        3/6
  Verifying        : cvmfs-fuse3-2.11.5-1.osg23.el8.x86_64                  4/6
  Verifying        : cvmfs-libs-2.12.0-1.el8.x86_64                         5/6
  Verifying        : cvmfs-libs-2.11.5-1.osg23.el8.x86_64                   6/6
Installed products updated.

Upgraded:
  cvmfs-2.12.0-1.el8.x86_64              cvmfs-fuse3-2.12.0-1.el8.x86_64
  cvmfs-libs-2.12.0-1.el8.x86_64

Complete!
```

**Comment by vvolkl** [comment]
Hmm, this may be related to #3707 , but I've never seen that in my testing. 

The postinstall reload is now done via a one-shot systemd service, to avoid possibly blocking the rpm transactions. You do have the service `/usr/lib/systemd/system/cvmfs-reload.service` correctly   installed after the update, right? Can you do a `systemctl start cvmfs-reload` and does it show up in the logs?

**Comment by DrDaveD** [comment]
Was that something installed with this rpm release?  Maybe it wasn't available yet?  Maybe it needed a `systemctl daemon-reload`?  I have
```
$ sudo systemctl status cvmfs-reload
* cvmfs-reload.service - Reload any cvmfs instances.
   Loaded: loaded (/usr/lib/systemd/system/cvmfs-reload.service; static; vendor>
   Active: inactive (dead)
$ sudo systemctl start cvmfs-reload
$ sudo systemctl status cvmfs-reload|cat
* cvmfs-reload.service - Reload any cvmfs instances.
   Loaded: loaded (/usr/lib/systemd/system/cvmfs-reload.service; static; vendor preset: disabled)
   Active: inactive (dead)

Dec 04 15:59:55 dwdosgdev.fnal.gov cvmfs_config[2088918]: dwd-config.opensciencegrid.org: Releasing saved glue buffer
Dec 04 15:59:55 dwdosgdev.fnal.gov cvmfs_config[2088918]: dwd-config.opensciencegrid.org: Releasing saved dentry tracker
Dec 04 15:59:55 dwdosgdev.fnal.gov cvmfs_config[2088918]: dwd-config.opensciencegrid.org: Releasing saved page cache entry cache
Dec 04 15:59:55 dwdosgdev.fnal.gov cvmfs_config[2088918]: dwd-config.opensciencegrid.org: Releasing chunk tables
Dec 04 15:59:55 dwdosgdev.fnal.gov cvmfs_config[2088918]: dwd-config.opensciencegrid.org: Releasing saved inode generation info
Dec 04 15:59:55 dwdosgdev.fnal.gov cvmfs_config[2088918]: dwd-config.opensciencegrid.org: Releasing fuse state
Dec 04 15:59:55 dwdosgdev.fnal.gov cvmfs_config[2088918]: dwd-config.opensciencegrid.org: Releasing saved open files table
Dec 04 15:59:55 dwdosgdev.fnal.gov cvmfs_config[2088918]: dwd-config.opensciencegrid.org: Releasing open files counter
Dec 04 15:59:55 dwdosgdev.fnal.gov cvmfs_config[2088918]: dwd-config.opensciencegrid.org: Activating Fuse module
Dec 04 15:59:55 dwdosgdev.fnal.gov systemd[1]: cvmfs-reload.service: Succeeded.
```

**Comment by DrDaveD** [comment]
Yes, if I do a `dnf downgrade cvmfs` the service is removed, and when I upgrade it I see the message again.

**Comment by DrDaveD** [comment]
I confirm this is fixed in 2.12.2.
